torch/csrc/utils/tensor_apply.cpp

```diff
@@ -78,6 +84,9 @@ Tensor & map_(Tensor & self, const Tensor & other_, PyObject* fn) {
 }
 
 Tensor & map2_(Tensor & self, const Tensor & x_, const Tensor & y_, PyObject* fn) {
+  if (self.is_meta()) {
+    return self;  // Just skip
+  }
```

Comment:
Do we care about meta performing accurate error reporting in the below cases, like when `x_` and `y_` have different dtypes? These `tensor_apply` methods seem like special snowflakes, so maybe we don't care as much about total meta/impl parity.

Reply:
Oh, I didn't even check this. Yes, you're right, ideally we would have matching error checking.